x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -123,13 +123,16 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
                 name: (
                   <>
                     {ALERTS_TAB}
-                    <ExperimentalBadge
-                      label={EXPERIMENTAL_LABEL}
-                      size="s"
-                      iconType="beaker"
-                      tooltipContent={EXPERIMENTAL_DESC}
-                      tooltipPosition="bottom"
-                    />
+                    {features.alerts.isExperimental ? (
```

Comment:
@cnasikas just curious: since `isExperimental` is optional and can be `undefined`, why did you need to add `isExperimental: false` explicitly in `x-pack/plugins/observability/public/pages/cases/cases.tsx`?

Reply:
If the `isExperimental` is `undefined` we defaulted to `false`. I add it `explicitly` in your file a) to notify your team, make you aware of the change and get your approval 🙂  b) be defined in case we change our defaults (pretty unlikely)